meeting-notes/2019/Q1/2019-01-07--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,183 @@
+    - Make CoreAPI tests not crash with go-ipfs-http-api - https://github.com/ipfs/go-ipfs/pull/5900 **(Needs Review)**
+ - Blocked:
+   - Mumble server setup waiting for infra response - https://github.com/protocol/ad-hoc-infra/pull/6
+   - (Decision needed) - Do we relaly need go/js core interfaces to live next to each other  https://github.com/ipfs/interface-ipfs-core/issues/423
+ - Next:
+   - Implement all major CoreAPI parts in go-ipfs-http-api
+   - Try to ressurect datastore benchmark efforts
+   - Setup mumble server (impromptu voice communication) - https://github.com/ipfs/go-ipfs/issues/5837
+     - Should I make it publicly accessible or should it have a password / something like that for now?
+
+@warpfork
+  - Done:
+    - Happy holidays, y'all
+    - IPLD: had some good conversations at CCC, got new ideas about open-vs-closed unions/etc; have notes, need to make them accessible to others.
+  - Blocked: nah
+  - Next:
+    - Boil open-vs-closed into a decision tree... um, schema, and document that.
+    - Moar go-ipld-prime implementation.  Maybe fleshing out some of the ipldcbor.Node impl?  A lot of things need love.
+
+@djdv
+ - Done:
+   - Minor documentation things
+   - this /ipfs/QmUiDVir9imgCpPLmXgTwLRTpHXrkn4vg2n7grc54MRk1N
+   - Large mount refactor, much better performance (https://github.com/ipfs/go-ipfs/issues/5003#issuecomment-449764627)
+   - mount now builds and runs on MacOS and Linux (dependencies are not gx'd yet)
+ - Next:
+   - Continue investigating showstopping elusive bug in mount and third party libraries it depends on (detailed https://github.com/ipfs/go-ipfs/issues/5003#issuecomment-451751681)
+   - Get pkg in a state where other people can help review/debug, coupled with a screencast walking through the implementation (sometime after lunch)
+
+@kevina 
+  - Done:
+    - Rebase `--cid-base=base32` 
+    - Work on config options for specifing default CID base
+  - Blocked/Asks:
+     - `--cid-base` p.r. needs help deciding command line option for upgrading CIDv0 to CIDv1 in order to display the CID in the correct multibase https://github.com/ipfs/go-ipfs/pull/5789, likely to go with `--force-cid-base` 
+  - Next:
+    - Today: Write status report for go side of CIDv1 base32 (keep forgetting about it, sorry) -- note Eric says no longer needed
+    - Hopefully get `--cid-base` p.r. merged
+    - Move config options for changing the default base issue along 
+    - Start work on chagning the DHT (providers) to use Multihashes instead of CIDs
+      https://github.com/libp2p/go-libp2p-routing/issues/32
+
+@stebalien
+  - Done:
+    - Email catchup
+    - Maintainer stuff
+    - Some work on CORs
+    - Debugged gateway issue
+  - Next:
+    - More catchup, review, maintainer stuff.
+    - Make sure the bitswap bug affecting the gateway gets fixed.
+    - Cut a release.
+   
+
+@bigs
+  - Done: 
+    - vacation!
+    - email catch up
+    - unblocking daemon progress
```

Comment:
@bigs could you be more descriptive here? What does it mean "unblocking daemon progress"? Is there any trail of breadcrumbs to issues and PRs?